### DISCUSSION TITLE
(UX) Please add "invert color" to Toolbar icons.

### DISCUSSION THREAD

**Discussion by gety9** [plateaukao/einkbro#430] [Ideas]
Title. That's one of the most frequent actions e-ink browsing user might want to peform, and right now we need to do 2 presses to achieve it. (and what's worse user often wants to go back to previous state (re-invert color), so he needs 4 presses :))

-, Also if possible add "copy link" to Toolbar icons.

Both actions are already in EinkBro, we just need to do "menu>action", instead of simply press "action" on toolbar.

**Comment by plateaukao** [comment]
@gety9 
Added. 0b8607d4
Will be avialable in next version (though it's not showing correct toggle state yet, but the state is very obvious on the web content).
<img width="368" alt="image" src="https://github.com/user-attachments/assets/f37ba60f-4fc0-4bc9-972f-6f605feffdfd">

**Comment by gety9** [comment]
@plateaukao Daniel, you are amazing! Thank you for listening for feedback and implementing all these new features and tweaks!
